packages/cli/src/commands/create-plugin/createPlugin.ts

```diff
@@ -221,20 +224,28 @@ export default async () => {
   }
 
   const answers: Answers = await inquirer.prompt(questions);
-
+  const name = cmd.scope
+    ? `@${cmd.scope.replace(/^@/, '')}/plugin-${answers.id}`
+    : `plugin-${answers.id}`;
+  const scopeName = cmd.scope ? `@${cmd.scope.replace(/^@/, '')}` : '';
+  const npmRegistry = cmd.npmRegistry && cmd.scope ? cmd.npmRegistry : '';
+  const privatePackage = cmd.private === false ? false : true;
+  const isMonoRepo = await fs.pathExists(paths.resolveTargetRoot('lerna.json'));
   const appPackage = paths.resolveTargetRoot('packages/app');
   const templateDir = paths.resolveOwn('templates/default-plugin');
   const tempDir = resolvePath(os.tmpdir(), answers.id);
-  const pluginDir = paths.resolveTargetRoot('plugins', answers.id);
+  const pluginDir = isMonoRepo
+    ? paths.resolveTargetRoot('plugins', answers.id)
+    : paths.resolveTargetRoot(answers.id);
   const ownerIds = parseOwnerIds(answers.owner);
-  const { version } = await fs.readJson(paths.resolveTargetRoot('lerna.json'));
+  const version = backstageVersion;
```

Comment:
Why was this changed?

Reply:
In another comment they mention `We can't extract version from lerna.json. Plugins will eventually have a different version from backstage. I'll use version from plugin's package.json`, could that be related?

Reply:
Ah, so reading back I'm guessing this was to make the diff work?

In that case, revert this line to the original, and grab the version from the target `package.json` itself during diff, instead of `lerna.json`

Reply:
during diff, it grabs from package.json. 
https://github.com/spotify/backstage/pull/2316/files#diff-8d46217384c7cb4b4188d65bc1341d83R87
 
By using `lerna.js` here, we can't create-plugin outside of monorepo.